client/src/components/share/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { ShareTemplate } from './share-template';
+import { useShare } from './use-share';
+
+interface ShareProps {
+  superBlock: string;
+  block: string;
+}
```

Comment:
```suggestion
import { useShare, type ShareProps } from './use-share';
```

Reply:
I removed ``ShareProps `` from ``'./use-share'`` and kept in index.tsx as the single point of ``ShareProps`` interface.

Hope it gets reusing everywhere